app.grayjay.Desktop.yaml

```diff
@@ -0,0 +1,85 @@
+app-id: app.grayjay.Desktop
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+
+command: Grayjay
+finish-args:
+  - --share=network
+  - --socket=pulseaudio  # We want to play sound, I suppose
```

Comment:
"I suppose" 

So you don't know?

Reply:
My apologies, this is my first major attempt at packaging a flatpak. 

I noticed that a very similar, existing flatpak app (freetube) uses this in its [manifest](https://github.com/flathub/io.freetubeapp.FreeTube/blob/0735b6da6dfda8e74e5b4827d9ad0a4efb662a72/io.freetubeapp.FreeTube.yml#L14) as well, so i figured, since this seems liek a relatively benign audio permission (as opposed to full filesystem or system bus access) i would err on the side of providing users a good experience, even if there are slightly extra (but not unreasonably so) permissions.

Flatpak users are expecting functioning, [stable](https://docs.flathub.org/docs/for-app-authors/requirements#stable-releases) software, are they not?

Reply:
yes, pulseaudio is in fact required to play audio. Removed the comment

Reply:
this has been done